src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -302,7 +302,9 @@ export default {
 			</template>
 
 			<!-- @slot The checkbox/radio label -->
-			<slot />
+			<template v-if="!!$slots.default" #default>
+				<slot />
+			</template>
```

Comment:
Not sure, but I'd expect `<!-- @slot` works only directly before `<slot>`

Reply:
Cannot check on Netlify 🙈

Reply:
Yeah, probably. Adjusted.